src/main/scala/nl/knaw/dans/easy/stage/dataset/Util.scala

```diff
@@ -67,6 +67,25 @@ object Util {
     } yield audience.text
   }
 
+  def readFileType(filePath: String)(implicit s: Settings): Try[Option[String]] = Try {
+    val filetype = for {
+      file <- loadBagXML("metadata/files.xml") \\ "files" \ "file"
+      if (file \ "@filepath").text == filePath
```

Comment:
you can do `file \@ "filepath"` here. The made `\@` a special operator for querying attributes.